protocols/imap/src/main/java/org/apache/james/imap/api/process/SelectedMailbox.java

```diff
@@ -89,6 +91,13 @@
      */
     MailboxPath getPath();
 
+    /**
+     * Return the mailboxId of the selected Mailbox.
+     *
+     * This is beneficial as the MailboxId is immutable.
```

Comment:
from the perspective of the API user, this sentence doesn't make much sense IMO.